src/core/network/client.ts

```diff
@@ -338,7 +344,7 @@ class ApiClient {
           options.attachmentAgent
         )
       : await buildJSONConfig(payload)
-    const apiUrl = `${options.apiRoot}/bot${token}/${method}`
+    const apiUrl = `${options.apiRoot}/${options.apiType}${token}/${method}`
```

Comment:
While the line is changed anyway, how about stripping handling trailing slash in `apiRoot` by
```suggestion
    const apiUrl = new URL(`${options.apiType}${token}/${method}`, options.apiRoot)
```

Reply:
I don't see much point in this. It looks like it still needs to be converted to a string because I think `fetch` takes a `string`.

Reply:
Incorrect. `fetch` accepts `URL` too.